frontend/src/services/createPostService.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/prefer-default-export
+export const submitPost = async postInfo => {
+  const parameters = {
+    ...postInfo,
```

Comment:
Might be clearer/more safe to create arguments for the post's parameters, i.e. somethinng like

```javascript
const submitPost = async (title, body) => {
    const parameters = {
        title,
        body,
        sender_created_at: //...
    }
    // ...
}
```

This prevents issues with e.g. passing incorrect parameter names into the function.